### PR TITLE
Add `view-markdown-source` feature

### LINKED ISSUE #946: Add ability to view and link to Markdown source
There's no way to view the source of Markdown files inside the GitHub UI. All you can do is click "Raw" to view the raw file.

A trick I discovered is to click "Blame". Then you can view the source of Markdown and even link to specific lines.

Would be nice to add a "Source" button when viewing a rendered Markdown file, which would go to the blame view, but add a query param that instructs Refined GitHub to tweak the UI, like removing the blame components, so it's a clean source view.

---

*For anyone that wants this, please also send this feature request to GitHub: support@github.com*

### PR DESCRIPTION
**Pull request by bfred-it**
Closes #946 

# Test

https://github.com/sindresorhus/refined-github/blob/master/readme.md

### CODE REVIEW

File: source/libs/fetch-dom.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+async function fetchDom(url: string): Promise<DocumentFragment>;
+async function fetchDom(url: string, selector: string): Promise<Element>;
+async function fetchDom(url: string, selector?: string): Promise<Node> {
+	const response = await fetch(url);
+	const dom = domify(await response.text());
+	return selector ? dom.querySelector(selector) : dom;
+}
+
+export default fetchDom;
```
Comment (bfred-it) [inline source/libs/fetch-dom.ts]: This can be reused in other parts of RGH too. `doma` is almost only used for this purpose in RGH.

**Comment by sindresorhus** [comment]
When I click the source button, the "Display the source blob" tooltip get's stuck and never disappears.

**Comment by sindresorhus** [comment]
Would be nice if the URL had line references, it would automatically go to the source view. Example: https://github.com/avajs/ava/blob/master/readme.md#L41-L51

**Comment by sindresorhus** [comment]
This feature is super useful!

**Comment by bfred-it** [comment]
Even `y-copy` works!

**Comment by JonathanTM** [comment]
I think this is broken by [recent changes](https://github.blog/changelog/2020-06-23-design-updates-to-repositories-and-github-ui/). Sad face.

**Comment by kidonng** [comment]
@JonathanTM #3282

**Pull request merged**